aws/aws-how-to/instances/create-ec2-using-ubuntu-image.rst

```diff
@@ -0,0 +1,89 @@
+Launch an Ubuntu-based EC2 instance
+===================================
+
+Running Ubuntu on Amazon Web Services requires you to go through the following steps: 
+
+1. Create AWS account
+2. Setup credentials
+3. Find an Ubuntu AMI
+4. Create a security group (Optional)
+5. Instantiate the image
+
+Create AWS account
+------------------
+
+If you don't have AWS account, go to https://aws.amazon.com, and select :guilabel:`Sign In to the Console` on the right top corner to create. Follow the instructions given in `Create a standard alone AWS account`_ to do so.
+
+After signing up, go to `EC2 console <https://console.aws.amazon.com/ec2/home>`_ and sign in with your account.
+
+Setup Credentials
+-----------------
+
+To use AWS and EC2 services, you need key pairs and access keys. Setup your credentials by following instructions from `EC2 credentials`_.
```

Comment:
change `EC2 credentials`_ to :doc:`../../aws-reference/ec2-credentials` (since this is not an external link and points to a doc within the same project)

Also include the actual instructions (last two sections of the reference page) here. Then write something like - To understand the different types of EC2 credentials better, refer to :doc:`../../aws-reference/ec2-credentials`